### PR TITLE
CAMEL-16852 - Moved ServiceKeys model to camel-huaweicloud-common

### PR DESCRIPTION
**Pull request by hokutor**
Move https://github.com/apache/camel/blob/main/components/camel-huawei/camel-huaweicloud-iam/src/main/java/org/apache/camel/component/huaweicloud/iam/models/ServiceKeys.java model to be common and shared between all the huawei cloud components. Currently there is a duplicated copy in all individual component modules.

Reference: Conversation at https://camel.zulipchat.com/#narrow/stream/257298-camel/topic/Common.20package.20for.20HuaweiCloud.20components 

CAMEL-16852 - Moved ServiceKeys model to camel-huaweicloud-common

### CODE REVIEW

File: docs/components/modules/ROOT/pages/hwcloud-iam-component.adoc [hunk 1]
```diff
@@ -1,7 +1,7 @@
 [[hwcloud-iam-component]]
 = Huawei Identity and Access Management (IAM) Component
 //THIS FILE IS COPIED: EDIT THE SOURCE FILE:
-:page-source: components/camel-huawei/camel-huaweicloud-iam/src/main/docs/hwcloud-iam-component.adoc
+:page-source: componentscamel-huaweicamel-huaweicloud-iamsrcmaindocshwcloud-iam-component.adoc
```
Comment (oscerd) [inline docs/components/modules/ROOT/pages/hwcloud-iam-component.adoc:4]: This is wrong, because you're on Windows I guess

**Review by oscerd — CHANGES_REQUESTED** [review]
The component seems to be missing from components/pom, also we need to check the docs generation in camel-package-maven-plugin, because this components set is an exception.

File: docs/components/modules/ROOT/pages/hwcloud-iam-component.adoc (hunk 1 above)
Comment (hokutor) [reply docs/components/modules/ROOT/pages/hwcloud-iam-component.adoc]: Have run the mvn routines from a mac/linux platform. Should look good  now

**Comment by hokutor** [comment]
> The component seems to be missing from components/pom, also we need to check the docs generation in camel-package-maven-plugin, because this components set is an exception.

Can you advice on the action to be done for this from my end?

**Comment by davsclaus** [comment]
Have you look in
https://github.com/apache/camel/blob/main/tooling/maven/camel-package-maven-plugin/src/main/java/org/apache/camel/maven/packaging/PrepareCatalogMojo.java

**Comment by hokutor** [comment]
> Have you look in
> https://github.com/apache/camel/blob/main/tooling/maven/camel-package-maven-plugin/src/main/java/org/apache/camel/maven/packaging/PrepareCatalogMojo.java

Perfect! thanks @davsclaus for pointing that. PrepareCatalogMojo updated

**Review by davsclaus — APPROVED** [review]

**Pull request merged**